tests/assertions/contains_test.rs

```diff
@@ -0,0 +1,44 @@
+use anyhow::Result;
+use k9::{assert_contains, assert_matches_snapshot};
+
+#[test]
+fn test_assert_contains() -> Result<()> {
+    super::setup_test_env();
+
+    assert!(assert_contains!(vec![1, 2, 3, 4, 5], 5).is_none());
+    assert!(assert_contains!(vec![9.8, 3.15], 2.5).is_some());
+    assert!(assert_contains!(vec![9.8, 3.15], 9.8).is_none());
+    assert!(assert_contains!(&[9.8, 3.15], &2.5).is_some());
+    assert!(assert_contains!(&[9.8, 3.15], &9.8).is_none());
+    assert!(assert_contains!(&[(1, 2), (2, 3)], &(3, 4)).is_some());
+    assert!(assert_contains!(&[(1, 2), (2, 3), (3, 4)], &(3, 4)).is_none());
```

Comment:
can you test `HashMap, BTreeMap, BTreeSet` and maybe some other non-collection iterable?

e.g.
```rust
assert_contains!("abc".chars(), 'a');
assert_contains!(Some(1), 1);
```

Reply:
also maybe slices and arrays?